index.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+<html lang="">
+
+<head>
+  <meta charset="utf-8">
+  <title></title>
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+</head>
+<body>
+<script src="./js/app.js"></script>
+<h2>Форма обратной связи</h2>
+<form method="post">
+  <p><label for="name">Имя:</label></p>
+  <p>
+    <input type="text" id="name" name="name" placeholder="Иван">
+  </p>
+  <p id="name_validation_result"></p>
```

Comment:
Используй camelCase вместо snakeCase для согласованности со стилем проекта

```suggestion
  <p id="nameValidationResult"></p>
```